### PR TITLE
Ensure non-artsy dependencies aren't automerged

### PR DESCRIPTION
**Pull request by zephraph**
After opening https://github.com/artsy/kaws/pull/160 I noticed that renovate was trying to automerge it. Generally, I think it's best practice that we _never_ automerge a non-artsy dependency unless we're doing it through merge on green. 

 
---

### Choose a release label

This repository uses [auto](https://github.com/intuit/auto) to generate releases. In order to do that, 
it needs an appropriate label assigned to each PR. Choose a label below that you feel best suites your changes.

##

<sub>**Semver Labels** _(choose at most one)_</sub>

- [ ]   
- [ ]   
- [ ]  

##

<sub>**Skip Release Labels** _(chose at most one)_</sub>

- [ ]   
- [x]   
- [ ]  

<sub>_Generated by [auto-it](https://github.com/apps/auto-it)_</sub>

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -2,9 +2,7 @@ const merge = require("deepmerge");
 
 const getCommitMessageExtraDefault = () => {
   const { getOptions } = require("renovate/dist/config/definitions");
-  const commitMessageExtra = getOptions().find(
-    opt => opt.name === "commitMessageExtra"
-  );
+  const commitMessageExtra = getOptions().find(opt => opt.name === "commitMessageExtra");
```
Comment (zephraph) [inline lib/config.js]: Prettier...

File: lib/config.js [hunk 2]
```diff
@@ -25,7 +23,8 @@ const issueApproval = {
       depTypeList: ["dependencies", "devDependencies", "peerDependencies"],
       packagePatterns: ["*"],
       excludePackagePatterns: ["^@artsy", "^@omakase", "^vscode-apollo-relay"],
-      masterIssueApproval: true
+      masterIssueApproval: true,
+      automerge: false
```
Comment (zephraph) [inline lib/config.js]: This was the only real change

File: lib/config.js [hunk 3]
```diff
@@ -47,12 +46,7 @@ const ignoreEngineUpdates = {
 const trivialLabel = {
   packageRules: [
     {
-      depTypeList: [
-        "dependencies",
-        "peerDependencies",
-        "devDependencies",
-        "orb"
-      ],
+      depTypeList: ["dependencies", "peerDependencies", "devDependencies", "orb"],
```
Comment (zephraph) [inline lib/config.js]: More prettier

**Comment by zephraph** [comment]
@pepopowitz I just added you to this for context. Let me know if you have questions.

**Comment by artsyit** [comment]
:rocket: PR was released in v1.17.1 :rocket:

**Pull request merged**